### ISSUE TITLE
how to create a hot voting-only wallet

### ISSUE THREAD

**Issue by raedah** [decred/dcrdocs#289]
raedah [10:40 AM] 
I don't think we have a section in the docs on doing a voting only setup.

new messages
davecgh [10:43 AM] 
Not sure, but I would guess because it's not really any different than a normal setup.  Just don't ever send any spendable coins to the wallet.  There is really nothing special or magical about it.  I mean to say there are no unique flags or anything. (edited)

jy-p [10:45 AM] 
still, i think it deserves some doco

davecgh [10:45 AM] 
Sure.  Stating the above in some doco couldn't hurt. (edited)

devnant [11:02 AM] 
by the way, could anyone give me a step by step on how to create hot voting-only wallets?

[11:02] 
using the cli

ay-p [11:03 AM] 
mmm just ./dcrwallet --create

[11:03] 
in some other directory

davecgh [11:03 AM] 
See above.  Do exactly the same thing you would do with a normal wallet, except just never send spendable coins to it.

ay-p [11:03 AM] 
right

davecgh [11:04 AM] 
In other words, don't import your existing seed.  Create a new one.

ay-p [11:04 AM] 
yup then just getnewaddress on the new wallet and then that's your ticket address for ticket purchasing wallet

**Comment by gratefulcheddar** [comment]
I believe @jzbz and gravityz3r0 are collaborating on this.

**Comment by jzbz** [comment]
Yeah, it's a work in progress, feedback is welcome...
https://github.com/jzbz/dcrdocs/blob/master/docs/mining/proof-of-stake/solo-proof-of-stake.md

**Comment by jolan** [comment]
This is a dupe of #187 which just didn't have a very good title.
